README.md

```
@@ -1,317 +1,225 @@
-@enduml
+```bash
+nix-on-droid switch --flake .#android
 ```
 
----
+## Repository Structure
 
-#### 🖼️ Imagen Prompt: "Happy Package Installation"
+-   `flake.nix`: The heart of the configuration. It defines all inputs, manages overlays, and exposes the final system and home configurations.
+-   `home/`: Contains modular Home Manager configurations for different package sets.
+    -   `base.nix`: Core utilities needed in all environments.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)